open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/common/factory/AcceleratorConsentExtensionFactory.java

```diff
@@ -65,47 +57,4 @@ public static ConsentManageRequestHandler getConsentManageRequestValidator(Strin
         return consentManageRequestHandler;
 
     }
-
-    /**
-     * Method to get the Consent Authorize Handler.
-     *
-     * @param type  Type of the request
-     * @return ConsentAuthorizeHandler
-     */
-    public static ConsentRetrievalHandler getConsentRetrievalHandler(String type) {
-
-        ConsentRetrievalHandler consentRetrieveHandler = null;
-
-        if (type.equalsIgnoreCase(ConsentExtensionConstants.ACCOUNTS)) {
-            consentRetrieveHandler = new AccountConsentRetrievalHandler();
-        } else if (type.equalsIgnoreCase(ConsentExtensionConstants.PAYMENTS)) {
-            consentRetrieveHandler = new PaymentConsentRetrievalHandler();
-        } else if (type.equalsIgnoreCase(ConsentExtensionConstants.FUNDSCONFIRMATIONS)) {
-            consentRetrieveHandler = new CofConsentRetrievalHandler();
-        }
-        return consentRetrieveHandler;
-
-    }
-
-    /**
-     * Method to get the Consent Persistence Handler.
-     *
-     * @param type  Type of the request
-     * @return ConsentPersistenceHandler
-     */
-    public static ConsentPersistenceHandler getConsentPersistenceHandler(String type) {
```

Comment:
shall we remove this in your persist flow PR?

Reply:
Fixed in [01f2f08]